README.md

```diff
@@ -8,3 +8,9 @@ Set of tutorials to use MLIPs:
 - [Equation of State](eos.ipynb) [![badge](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/stfc/janus-tutorials/blob/main/eos.ipynb)
 - [Phonons](phonons.ipynb) [![badge](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/stfc/janus-tutorials/blob/main/phonons.ipynb)
 
+
+Tutorials to use MLIPs through AiiDA
```

Comment:
This could maybe be elaborated on slightly to explain how they differ from the tutorials above?

It's not the only thing but, for example, I'm guessing these are slightly more sequential/at least the first is required before the others can be run, which isn't immediately obvious.

Reply:
Also more highlighted than newly added as an issue, but we should probably explicitly link janus-core and aiida-mlip for more context/docs